Added Healing Rain, Downpour and Wellspring to Unleash Life buff usage breakdown

Fassbrause opened the pull request:
I added 3 new colors in in the resto shaman constants for Healing Rain, Downpour and Wellspring, check whether these seem appropriate to you guys.

trevorm4 reviewed (changes requested):
Re-add the .env.local file and can you add an image to your PR to show an example of what it looks like with the new spells?

Otherwise it looks good overall

Fassbrause commented:
Here's how it looks:
![image](https://user-images.githubusercontent.com/121354216/209677938-d4a8234b-8129-4185-86e2-a5765389ee34.png)
My bad, I renamed env.local.example instead of copying it.

Fixed now!
